packages/schema/src/dialects/postgres.ts

```diff
@@ -46,9 +62,15 @@ export default class Postgres extends KnexPostgres implements SchemaInspector {
       `,
 				[this.explodedSchema.join(',')]
 			),
-			this.knex
-				.raw(
-					`
+		]);
+
+		const columns: RawColumn[] = columnsResult.rows;
+		const primaryKeys = primaryKeysResult.rows;
+		const geometryColumns: RawGeometryColumn[] = await new Promise((resolve) => {
+			if (!columns.some((col) => col.table_name === 'geometry_columns')) return resolve([]);
```

Comment:
"geometry_columns" is a view, not a table. I don't think it shows up under `columns` at all. We'll have to find an alternative way to find whether or not PostGIS is available

Reply:
We might be able to rely on 

```sql
SELECT EXISTS (
	SELECT
	FROM
		pg_catalog.pg_class c
		JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
	WHERE
		n.nspname = 'public'
		AND c.relname = 'geometry_columns'
		AND c.relkind = 'v' -- only views
);
```

to check if the view exists before continuing (cc @Oreilles)